refarch-backend/src/main/resources/logback-spring.xml

```diff
@@ -49,35 +49,20 @@
 
 
     <springProfile name="json-logging">
-
-        <!-- Log appender -->
-        <appender name="JSON_STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-            <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
-                <providers>
-                    <!-- provides the fields in the configured pattern -->
-                    <pattern>
-                        <!-- the pattern that defines what to include -->
-                        <pattern>
-                            {
-                                "timestamp" : "%date{yyyy-MM-dd'T'HH:mm:ss.SSS}",
-                                "appName" : "${springAppName}",
-                                "TraceId" : "%mdc{traceId}",
-                                "SpanId" : "%mdc{spanId}",
```

Comment:
What about the trace and span ID? If i am running it locally, i cant see any output for these two.

Reply:
Everything that is put in the "Mapped Diagnostic Context" (MDC) of the Logger gets included (see `LogbackJsonLoggingConfigurationTest.puts_mdc_in_json`).

Also tested it running locally, as long as a log message is procuded (and logged) in context of a request, there are spanId and traceId included:

```json
{"@timestamp":"2024-09-04T11:45:28.1479043+02:00","@version":"1","message":"Completed 404 NOT_FOUND","logger_name":"org.springframework.web.servlet.DispatcherServlet","thread_name":"http-nio-8080-exec-1","level":"DEBUG","level_value":10000,"springAppName":"refarch-eai","traceId":"66d82c381d2bab4f1f1a1b8d717d7e09","spanId":"1f1a1b8d717d7e09"}
```

Currently `!json-logging` / `json-logging` Profiles configure different Logger log levels, e.g. only on `!json-logging` the DEBUG log level is set on a couple of framework classes like `org.springframework.web`. That part of configuration I left "as-is" - maybe this needs a change?